cve_scripts/paths2json.py

```diff
@@ -0,0 +1,49 @@
+import sys, getopt, os, json
+
+#Add the pathname fo the file that contains the pop paths data on your local machine
+#pathname = 
+filename = os.path.basename(pathname)
+
+path_lines = {}
+result = ""
+with open(filename) as file:
+    for line in file:
+        stripped_line = line.strip()
+        if stripped_line.startswith("SF"):
```

Comment:
did we not find a better way to deal with this instead of this massive elif chain?

Reply:
try a switch statement maybe? or a for loop with an array of the options which breaks once one of the options is met?

Reply:
Okay, elif chain has been removed in latest commit with a for loop over an array of the file path prefixes